rpm/sailfish-utilities.spec

```diff
@@ -23,8 +23,9 @@ BuildRequires: qt5-default
 BuildRequires: qt5-qttools
 BuildRequires: qt5-qttools-linguist
 %if %{with l10n}
-%define _all_translations_version %(rpm -q --queryformat "%%{version}-%%{release}" %{name}-all-translations)
 BuildRequires: %{name}-all-translations
+#!BuildIgnore: %{name}-all-translations-pack
+%define _all_translations_version %(rpm -q --queryformat "%%{version}-%%{release}" %{name}-all-translations)
```

Comment:
sailfish-utilities-l10n-<lang> packages are provided by both sailfish-utilities-all-translations and sailfish-utilities-all-translation-pack. So to make the build work Ignore the pack one